vendor/github.com/moby/buildkit/cache/remotecache/gha/gha.go

```diff
@@ -140,8 +141,9 @@ type exporter struct {
 func NewExporter(c *Config) (remotecache.Exporter, error) {
 	cc := v1.NewCacheChains()
 	cache, err := actionscache.New(c.Token, c.URL, c.Version > 1, actionscache.Opt{
-		Client:  tracing.DefaultClient,
-		Timeout: c.Timeout,
+		Client:    tracing.DefaultClient,
+		Timeout:   c.Timeout,
+		UserAgent: bkversion.UserAgent(),
```

Comment:
Saw the user agent was added; does the user agent and/or should it be matching "docker engine" or "buildkit"? @jsternberg @tonistiigi

Reply:
This is the same user-agent used for registry requests and `ADD`. The value is configurable by the importer (but unrelated to this PR).

Reply:
Ah, thanks; wasn't sure if it's something we had do wire up to pass through our user-agent.


(but perhaps something for a future release)